redis/redis.go

```diff
@@ -1,7 +1,11 @@
 package redis
 
 import (
-	"github.com/go-redis/redis"
+	"context"
+	"time"
+
+	"github.com/redis/go-redis/extra/redisotel/v9"
+	"github.com/redis/go-redis/v9"
```

Comment:
With these import lines the `go.mod` is out of sync and when you run `go get ./...` or `go mod tidy` this will add the v9 to the `go.mod` file and effectively include the dependency update in this PR.

Your plan was probably to have the other separate [update PR](https://github.com/philippgille/gokv/pull/130) merged first and then this one, but this PR's second commit contains a *removal* of the dependency lines from the `go.mod`.

I would suggest to treat this PR as a "stacked PR" building entirely on top of the other one, without removing anything from the other one. For any new commit you might add onto the other, you can rebase this branch on the other to include all the same commits. Then when the other PR is merged, this PR's diff should be automatically changed to be only the relevant ones (either out of the box, or after squashing the commits that are originally from the other branch).

Reply:
make sense, thanks